test/TEST-24-UNPRIVILEGED/testsuite.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+#set -ex
+#set -o pipefail
+
+for i in /usr/lib/systemd/tests/test-*; do
```

Comment:
I guess this is where the driver pointed out in https://github.com/systemd/systemd/issues/10069 would come in handy.